### PR TITLE
Feature/release prep

### PR DESCRIPTION
**Pull request by jpaidoussi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,15 @@ services:
   - postgresql
 install:
   - "pip install -e ."
-  - "pip install -r requirements-dev.txt
+  - "pip install -r requirements-dev.txt --use-wheel"
```
Comment (JayH5) [inline .travis.yml:11]: You shouldn't need this or the `"` quotes. I guess you are just making this like the others, though.

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-pytest
-pytest-cov
-pytest-django
+pytest==2.8.5
```
Comment (JayH5) [inline requirements-dev.txt:1]: Any reason we're pinning to an old version of pytest? Just making this like the others?

**Review by JayH5 — APPROVED** [review]
:+1:

File: .travis.yml (hunk 1 above)
Comment (jpaidoussi) [reply .travis.yml:11]: The main problem was the missing closing quote which wasn't valid, so I just pasted a working line from another project.

File: requirements-dev.txt (hunk 2 above)
Comment (jpaidoussi) [reply requirements-dev.txt:1]: Yes, standardizing on how the others are setup now and will get to the latest version everywhere for v1.0

**Pull request merged**